src/utils/returnHeaderName.m

```diff
@@ -2,25 +2,20 @@
 
 function headerName = returnHeaderName(columnName, nbCol, iCol)
     %
-    % Short description of what the function does goes here.
+    % It return one by one the column name to be used as a header in a fresh opened event file
     %
     % USAGE::
     %
     %   [argout1, argout2] = templateFunction(argin1, [argin2 == default,] [argin3])
     %
```

Comment:
don't you want to delete this part?

Reply:
I think Marco will open a separate PR for this. One PR for the code and one for the DOC. #DivisionOfLabor.